pywbem_mock/_instancewriteprovider.py

```diff
@@ -277,114 +220,117 @@ def __init__(self, cimrepository=None):
 
     ####################################################################
     #
-    #   CreateInstance server request
+    #   CreateInstance responder
     #
     ####################################################################
 
     def CreateInstance(self, namespace, NewInstance):
         """
-        Implements a WBEM server responder for
+        Implements a mock WBEM server responder for
         :meth:`pywbem.WBEMConnection.CreateInstance`.
 
-        Create a CIM instance in the local CIM repository of this class.
+        Create a new CIM instance in the CIM repository of the mock WBEM server.
 
-        The input parameters must have been validated before this method
-        is called including:
+        Validation that already has been performed by the caller:
+        - The Python types of all parameters will be as specified.
+        - The namespace exists in the CIM repository.
+        - The creation class of the new instance exists in the namespace
+          in the CIM repository.
+        - All properties specified in the new instance are exposed by the
```

Comment:
why use exposed rather than "in"

Reply:
We use "exposed" in the DMTF specs, and it means "defined + inherited" elements.